src/site/_includes/partials/head.njk

```diff
@@ -32,15 +32,37 @@
 <link rel="icon" type="image/png" sizes="16x16" href="/images/favicon-16x16.png">
 <link rel="apple-touch-icon" sizes="180x180" href="/images/apple-touch-icon.png">
 <link rel="mask-icon" color="#0054ff" href="/images/safari-pinned-tab.svg">
-<script type="module" src="{{ helpers.hashForProd('/js/app.js') }}"></script>
+{% set scriptLoader %}
```

Comment:
It might be good to have a big comment here explaining what is going on and why we're doing it.